dom.go

```diff
@@ -1080,16 +1087,21 @@ func (w *window) Stop() {
 
 // TODO reuse util.EventTarget
 
-func (w *window) AddEventListener(typ string, useCapture bool, listener func(Event)) func(o *js.Object) {
-	wrapper := func(o *js.Object) { listener(wrapEvent(o)) }
-	w.Call("addEventListener", typ, wrapper, useCapture)
-	return wrapper
+func (w *window) RequestAnimationFrame(callback func()) func() {
+	w.Call("requestAnimationFrame", callback)
```

Comment:
requestAnimationFrame returns an integer, which we should return as well. There's no point in returning the callback.